src/components/Editor.singleton.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Get instance of Editor component
+ * @return {Promise<*>}
+ */
+export default async function getEditorInstance() {
+	if (!window.EditorInstance) {
+		if (window.EditorInstanceImporting) {
+			await new Promise((resolve) => setTimeout(resolve, 500))
```

Comment:
Feels a bit risky to wait for a timeout here, with the low value it could fail on slow network loading the editor in the other entrypoint, higher would delay loading of the editor. Maybe we can check more frequently in an inverval instead?